examples/sglang/utils/protocol.py

```
@@ -61,3 +61,18 @@ class DisaggPreprocessedRequest(BaseModel):
     bootstrap_port: int
     bootstrap_room: int
     data_parallel_rank: Optional[int] = None
+
+
+EmbeddingInput = Union[str, List[str], List[int], List[List[int]]]
+
+EncodingFormat = Literal["float", "base64"]
+
+
+class EmbeddingRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)